### PR TITLE
doc: Server EnableContextTimeout

### PR DESCRIPTION
**Pull request by felix021**
#### What type of PR is this?

doc

#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

File: content/zh/docs/kitex/Tutorials/service-governance/timeout.md [hunk 1]
```diff
@@ -151,20 +151,72 @@ rpctimeout.SetBusinessTimeoutThreshold(10 * time.Millisecond)
...
+svr := yourservice.NewServer(handler,
     server.WithExitWaitTime(5 * time.Second),
 )
 ```
-注：两个配置项可以按需独立配置。
+
+###### WithEnableContextTimeout
+
+Client
+- 指定 Transport 为 TTHeader
```
Comment (YangruiEmma) [inline content/zh/docs/kitex/Tutorials/service-governance/timeout.md:199]: 指定 Transport Protocol 为 TTHeader

File: content/zh/docs/kitex/Tutorials/service-governance/timeout.md [hunk 2]
```diff
@@ -151,20 +151,72 @@ rpctimeout.SetBusinessTimeoutThreshold(10 * time.Millisecond)
...
+```go
+import "github.com/cloudwego/kitex/server"
+
+svr := yourservice.NewServer(handler,
     server.WithExitWaitTime(5 * time.Second),
 )
 ```
-注：两个配置项可以按需独立配置。
+
+###### WithEnableContextTimeout
```
Comment (YangruiEmma) [inline content/zh/docs/kitex/Tutorials/service-governance/timeout.md:196]: 加一句，需结合 TTHeader 传输协议使用

File: content/zh/docs/kitex/Tutorials/service-governance/timeout.md (hunk 1 above)
Comment (YangruiEmma) [reply content/zh/docs/kitex/Tutorials/service-governance/timeout.md:199]: 也加一下 TTHeader 的配置方式吧

Comment (felix021) [reply content/zh/docs/kitex/Tutorials/service-governance/timeout.md:199]: 后面用法里有写配置方式

File: content/zh/docs/kitex/Tutorials/service-governance/timeout.md (hunk 2 above)
Comment (felix021) [reply content/zh/docs/kitex/Tutorials/service-governance/timeout.md:196]: 后面不是都写了吗

Comment (felix021) [reply content/zh/docs/kitex/Tutorials/service-governance/timeout.md:198]: done

File: content/zh/docs/kitex/Tutorials/service-governance/timeout.md (hunk 1 above)
Comment (felix021) [reply content/zh/docs/kitex/Tutorials/service-governance/timeout.md]: done

**Pull request merged**